apps/core/templatetags/core_tags.py

```diff
@@ -20,3 +20,7 @@ def navigation(context):
         "current_page": context.get("page"),
         "annotated_list": Page.get_annotated_list_qs(pages),
     }
+
+@register.filter
+def canonical_url(value, arg):
```

Comment:
Let’s rename this to something else like `hreflang_url` to reflect the fact this isn’t meant for canonical links.